src/python/ddapp/ikplanner.py

```diff
@@ -219,6 +220,7 @@ def __init__(self, ikServer, robotModel, jointController, handModels):
         self.additionalTimeSamples = 0
         self.useQuasiStaticConstraint = True
         self.pushToMatlab = True
+	self.pushToMatlabIK = True
```

Comment:
i think these flag could be renamed.  you've introduced a second flag and now it's not too clear what the intention is.